worlds/yachtdice/Locations.py

```diff
@@ -0,0 +1,67 @@
+from BaseClasses import Location
+import typing
+
+class LocData(typing.NamedTuple):
+    id: int
+    region: str
+    score: int
+
+class YachtDiceLocation(Location):
+    game: str = "Yacht Dice"
+
+    def __init__(self, player: int, name: str, score: int, address: typing.Optional[int], parent):
+        super().__init__(player, name, address, parent)
+        self.yacht_dice_score = score
+        self.event = not address
```

Comment:
What is this doing? I don't see you using it anywhere and it is no longer a property of `Location`

Reply:
Yup, thanks, I indeed don't use the event.
I've removed the `self.event = not address` line